### DISCUSSION TITLE
SSR disable for app router

### DISCUSSION THREAD

**Discussion by Heet-Bhalodiya** [vercel/next.js#61843] [Help]
### Summary

Can we turn off server-side rendering (SSR) for the entire app in Next.js 14 using the app router? Since the `target: serverless` option has been removed from the next.config.js file, and while I know about the `react-no-ssr` method, it only works for individual pages, not the whole app. Is there a simpler way to disable SSR for the entire app at once? Thank you.

**Comment by poojadosad** [comment]
I am wondering for the same, can we do SSR false for the whole application?
